### PR TITLE
FWI netcdf load and processing

### PR DESCRIPTION
**Pull request by miltminz**
New features in `datasets`:
- `fwi_utils.py`
   - load FWI data, unzip it and put it in `.data/` directory
   - get dataframe with longitude and latitude coordinates of netcdf files and French departments
- `fwi.py`: loader class for pandas dataframe with FWI indices (8 km resolution) for selected time period

Minor changes:
- `.gitignore`
- `config`: add paths to FWI zip files, repo and .data paths

### CODE REVIEW

**Comment by frgfm** [comment]
Thanks for the PR!
Quick comments:
- if fwi_utils functions are to be used only by fwi, perhaps we can put everything in fwi?
- the coverage diff seems to point out that we need to add some unittests

**Comment by miltminz** [comment]
Thanks for the comments. Indeed at the beginning everything was in one `fwi.py`. Functions in `fwi_utils` load and write data that is then used by the class in fwi.py so it seemed to me that it was better to split them, but actually it's true, it should be better to unify them again.
And for the coverage you're right as well, it was too late for today :-)

**Comment by chloeskt** [comment]
Hello @miltminz, all good for me :) you just need to add some unittests to improve the coverage

**Comment by miltminz** [comment]
Hello people, I made a couple of modifications
- I erased `fwi_utils.py`, only `fwi.py` remains. 
- Unit tests have been added
- Some refactoring + dependencies in `requirements.txt`

**Comment by frgfm** [comment]
@miltminz Would you mind merging master into this branch please? (PR #4 changes a few files you are modifying here, hence the conflicts)

**Comment by miltminz** [comment]
Yep, I'll do this later tonight, sorry I forgot

Le dim. 15 nov. 2020 à 13:15, F-G Fernandez <notifications@github.com> a
écrit :

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 pandas>=1.0.0
 geopandas>=0.8.1
 Rtree>=0.9.4
+shapely>=1.7.1
+netCDF4>=1.5.4
```
Comment (frgfm) [inline requirements.txt:5]: You'll need to update the setup.py file as well : https://github.com/pyronear/pyronear-datascience/blob/master/setup.py#L37 ;)

Comment (miltminz) [reply requirements.txt:5]: yep

**Review by chloeskt — APPROVED** [review]
Seems good to me now :)

**Pull request merged**